cttportfolio.css

```diff
@@ -467,4 +540,194 @@ footer{
 	max-height:100%;
```

Comment:
```suggestion
	max-height:100%;
```
why do you have this?

Reply:
So the imagewon't by default be it's normal height and width and it will be contained in the div